src/main/java/org/terasology/manualLabor/components/ShearableComponent.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.manualLabor.components;
+
+import org.terasology.engine.entitySystem.Component;
+
+/**
+ * Attached to components that can be sheared.
+ */
+public class ShearableComponent implements Component {
+
+    /**
+     * Stores the last time the shearing took place. Default -1 for no previous shearing event.
+     */
+    public long lastShearingTimestamp = -1;
+
+    /**
+     * Stores the current state of shearing i.e. whether sheared or not.
+     */
+    public boolean sheared;
```

Comment:
This might be just be my problem, but wherever I see this being used I have to think about what this is. Could we rename it to something that would signify that it is obviously boolean like isSheared?